### PR TITLE
RHDH pre-submit and post-submit job for 1.2.x

### PR DESCRIPTION
**Pull request by subhashkhileri**
https://issues.redhat.com/browse/RHIDP-2049

### CODE REVIEW

File: ci-operator/step-registry/janus-idp/backstage-showcase/1.2.x/janus-idp-backstage-showcase-1.2.x-ref.metadata.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+			"subhashkhileri"
+		],
+		"reviewers": [
+			"albarbaro",
+			"josephca",
+			"rnapoles-rh",
+			"subhashkhileri"
+		]
+	}
+}
```
[the remark this answers is not in the document]
Comment (subhashkhileri) [reply ci-operator/step-registry/janus-idp/backstage-showcase/1.2.x/janus-idp-backstage-showcase-1.2.x-ref.metadata.json:16]: @josephca This is auto-generated metadata, making any changes might fail the ci check on this job.

File: ci-operator/step-registry/janus-idp/backstage-showcase/1.2.x/nightly/janus-idp-backstage-showcase-1.2.x-nightly-ref.metadata.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+			"subhashkhileri"
+		],
+		"reviewers": [
+			"albarbaro",
+			"josephca",
+			"rnapoles-rh",
+			"subhashkhileri"
+		]
+	}
+}
```
[the remark this answers is not in the document]
Comment (subhashkhileri) [reply ci-operator/step-registry/janus-idp/backstage-showcase/1.2.x/nightly/janus-idp-backstage-showcase-1.2.x-nightly-ref.metadata.json:16]: @josephca This is auto-generated metadata, making any changes might fail the ci check on this job.

**Review by josephca — APPROVED** [review]
/lgtm
/approve

**Pull request merged**